Mathlib/Data/Finset/Sort.lean

```diff
@@ -76,6 +76,10 @@ theorem sort_insert [DecidableEq α] {a : α} {s : Finset α} (h₁ : ∀ b ∈
     sort r (insert a s) = a :: sort r s := by
   rw [← cons_eq_insert _ _ h₂, sort_cons r h₁]
 
+@[simp]
+theorem sort_range (n : ℕ) : sort (· ≤ ·) (range n) = List.range n :=
```

Comment:
Could you add the `<` version here and throughout too?

Reply:
The definition of `Multiset.sort` requires a total linear order, which `<` is not one of.